Languages/es.xml

```diff
@@ -819,91 +820,91 @@ Gracias a: Oleada, Jhovany200, nomemolesteis@gmail.com por la traducción.
 
     <String Target="隐藏资源管理器导航窗口中的网络" Link="Ocultar la ventana de navegación del explorador de red"/>
 
-    <String Target="优化非活动窗口标题栏颜色（by IT之家）" Link="Optimizar el color de la barra del título de las ventanas inactivas (by IT Home)"/>
+    <String Target="优化非活动窗口标题栏颜色（by IT之家）" Link="Optimizar el color de la barra del título de las ventanas inactivas (por IT Home)"/>
 
-    <String Target="关闭视频文件预览，提高资源管理器响应速度（by ‍莫失莫忘）" Link="Desactivar previo de vídeo para mejorar respuesta Windows Explorer (by MoWangMoShi)"/>
-    <String Target="关闭音乐文件图片预览，提高资源管理器响应速度（by ‍莫失莫忘）" Link="Desactivar previo de audio para mejorar respuesta Windows Explorer (by MoWangMoShi)"/>
-    <String Target="关闭打开程序的“安全警告”（by 莫失莫忘）" Link="Desactivar la advertencia de seguridad al abrir programas (by MoWangMoShi) "/>
-    <String Target="关闭在应用商店中查找关联应用（by Asoft）" Link="Desactivar la búsqueda de aplicaciones asociadas de Windows Store (by Asoft)"/>
-    <String Target="隐藏此电脑中视频、图片、文档、下载、音乐、桌面、3D对象七个文件夹（by 、Cloud.）" Link="Ocultar vídeos, imágenes, documentos, descargas, música, escritorio y 3D(by Cloud)"/>
-    <String Target="禁止自动安装推荐的应用程序（by IT之家）" Link="Se recomienda desactivar la instalación de las aplicaciones automáticamente (by ITHome)"/>
-    <String Target="关闭锁屏时的Windows 聚焦推广（by 、Cloud.）" Link="Desactivar Windows Focus en bloqueo de pantalla (by Cloud)"/>
-    <String Target="关闭“使用Windows时获取技巧和建议”（by 、Cloud.）" Link="Desactivar &quot;Obtener consejos al utilizar Windows&quot; (by Cloud)"/>
-    <String Target="隐藏任务栏上的人脉（by 、Cloud.）" Link="Ocultar la barra de tareas (by Cloud)"/>
+    <String Target="关闭视频文件预览，提高资源管理器响应速度（by ‍莫失莫忘）" Link="Desactivar previo de vídeo para mejorar respuesta explorador (por MoWangMoShi)"/>
+    <String Target="关闭音乐文件图片预览，提高资源管理器响应速度（by ‍莫失莫忘）" Link="Desactivar previo de audio para mejorar respuesta explorador (por MoWangMoShi)"/>
+    <String Target="关闭打开程序的“安全警告”（by 莫失莫忘）" Link="Desactivar la advertencia de seguridad al abrir programas (por MoWangMoShi) "/>
+    <String Target="关闭在应用商店中查找关联应用（by Asoft）" Link="Desactivar la búsqueda de aplicaciones asociadas de Windows Store (por Asoft)"/>
+    <String Target="隐藏此电脑中视频、图片、文档、下载、音乐、桌面、3D对象七个文件夹（by 、Cloud.）" Link="Ocultar carpetas: Objetos 3D, Escritorio, Documentos, Descargas, etc. (por Cloud)"/>
```

Comment:
Same as line 761.

Reply:
the majority of people know what follows, "etc" is in Spanish means to continue.......
since it does not enter everything.